Orion.Net/Controllers/BaseDataController.cs

```diff
@@ -65,8 +65,10 @@ public string Get(string id)
         [HttpPost]
         public void Post([FromBody]T model)
         {
+            var request = Request;
+            bool verification = request.Headers.ContainsKey("Authorization") ? (request.Headers["Authorization"] ==SupportID) : false;
```

Comment:
Please move this check in an Authorization Attribute.
More informations : https://docs.microsoft.com/fr-fr/aspnet/core/security/authorization/policies?view=aspnetcore-3.1